ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-presubmits.yaml

```diff
@@ -32,6 +32,6 @@ presubmits:
     agent: jenkins
     labels:
       master: ci.openshift.redhat.com
-    context: ci/openshift-jenkins
+    context: ci/jenkins/e2e
```

Comment:
Let's do `ci/jenkins/mao-e2e`

Reply:
that name will only show up in mao repo, so why add `mao`?

Reply:
Later one we can have more e2e tests. Let's make clear it's for the mao.

Reply:
Don't get it. Later we might have more e2e tests in mao repository which won't include mao? Strange.

Reply:
let's do it later then